asset-properties-widget/common/asset-property-constant.ts

```diff
@@ -0,0 +1,9 @@
+export const defaultProperty = [
+{ c8y_JsonSchema:{properties :{name:{type: "string",label: "Name"}} }, name: 'name', label: "Name", type: "string", active: true, isEditable:true, isExistingProperty:true},
```

Comment:
Won't isEditable should be false?

Reply:
we are allowed to edit name here as in DTM support to edit asset name.